### PR TITLE
Feature/tmb 5 Otp verification Page

### PR DESCRIPTION
**Pull request by S1ncer3ly**

### CODE REVIEW

File: lib/app/data/values/strings.dart [hunk 1]
```diff
@@ -5,7 +5,9 @@ class Strings {
   static const empty = '';
   static const mobileNumber = 'Mobile Number';
   static const getOTP = 'Get OTP';
+  static const verifyOtp = 'Verify Otp';
   static const retry = 'Retry';
+  static const didntGotACode ="Didn't get a code?";
```
Comment (istutiagarwal) [inline lib/app/data/values/strings.dart:10]: spell check

File: lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart [hunk 2]
```diff
@@ -1,21 +1,117 @@
...
-        title: Text('AuthVerifyOtpView'),
-        centerTitle: true,
-      ),
-      body: Center(
-        child: Text(
-          'AuthVerifyOtpView is working',
-          style: TextStyle(fontSize: 20),
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.fromLTRB(30, 70, 30, 0),
```
Comment (istutiagarwal) [inline lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart:20]: dimens?

File: lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart [hunk 3]
```diff
@@ -1,21 +1,117 @@
...
+          child: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              GestureDetector(
```
Comment (istutiagarwal) [inline lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart:24]: Y do we need gesture detector here

File: lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart [hunk 4]
```diff
@@ -1,21 +1,117 @@
...
+                  children: [
+                    Flexible(
+                      flex: 1,
+                      child: CircularContainerWithImage(
+                        icon: Images.ic_Shield,
+                        borderColor: AppColors.borderGrey,
+                        iconSize: 40,
+                      ),
+                    ),
+                    SizedBox(width: 15),
```
Comment (istutiagarwal) [inline lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart:38]: dimens

File: lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart [hunk 5]
```diff
@@ -1,21 +1,117 @@
...
+          child: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              GestureDetector(
+                onTap: () {
+                  // debugPrint("Row tapped!");
+                },
+                child: Row(
+                  children: [
+                    Flexible(
```
Comment (istutiagarwal) [inline lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart:30]: y r we using flexible here?

File: lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart [hunk 6]
```diff
@@ -1,21 +1,117 @@
...
+                        iconSize: 40,
+                      ),
+                    ),
+                    SizedBox(width: 15),
+                    Flexible(
+                      fit: FlexFit.tight,
+                      flex: 6,
+                      child: Text(
+                        'Enter Your verification Code',
+                        style: TextStyle(fontSize: 32, fontFamily: 'rozha one'),
```
Comment (istutiagarwal) [inline lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart:44]: use Styles file

File: lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart [hunk 7]
```diff
@@ -1,21 +1,117 @@
...
+                  return Expanded(
+                    child: TextField(
+                      controller: controller.controllers[index],
+                      focusNode: controller.focusNodes[index],
+                      inputFormatters: [
+                        FilteringTextInputFormatter.allow(RegExp(r'[0-9]')),
+                        LengthLimitingTextInputFormatter(1),
+                      ],
+                      onChanged: (value) {
+                        if (value.length == 1) {
```
Comment (istutiagarwal) [inline lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart:77]: move this to controller

File: lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart (hunk 5 above)
Comment (S1ncer3ly) [reply lib/app/modules/auth/verify-otp/views/auth_verify_otp_view.dart:30]: for responsiveness its need or we are getting a pixel overflow

**Pull request merged**